Presentation.Web/app/shared/selectOption/selectOption.directive.ts

```diff
@@ -15,7 +15,7 @@
                     autoSaveUrl: "@",
                     appendurl: "@",
                     field: "@",
-                    disabled: "&ngDisabled",
+                    disabled: "=ngDisabled",
```

Comment:
Hvorfor skal den ændres fra "%" til "="?

Reply:
Det har noget med scope at gøre for at data binding rent faktisk virker